Source/Fuse.Controls.Panels/Layouts/WrapLayout.uno

```diff
@@ -183,10 +183,9 @@ namespace Fuse.Layouts
 				var saMin = IsVert ? AlignmentHelpers.GetHorizontalSimpleAlignOptional(RowAlignment) : AlignmentHelpers.GetVerticalSimpleAlignOptional(RowAlignment);
 				var saMaj = IsVert ? AlignmentHelpers.GetVerticalSimpleAlignOptional(RowAlignment) : AlignmentHelpers.GetHorizontalSimpleAlignOptional(RowAlignment);
 				var elp = lp.CloneAndDerive();
-				for (int i=0; i < elements.Count; ++i)
+				i = 0;
+				for (var element = container.FirstChild<Visual>(); element != null; element = element.NextSibling<Visual>(), i++)
```

Comment:
Same here, not the same `i` value  (if this change was consistent it might work, but it feels safer to iteratre over `Node` like before).